pkg/stats/stats.go

```diff
@@ -132,12 +134,13 @@ func InstantiateNewStats(playerOneId string, playerTwoId string) *entity.Stats {
 		NotableData:   instantiateNotableData()}
 }
 
-func AddGame(stats *entity.Stats, lss ListStatStore, history *pb.GameHistory, req *ipc.GameRequest,
+func AddGame(ctx context.Context, stats *entity.Stats, lss ListStatStore, history *pb.GameHistory, req *ipc.GameRequest,
 	cfg *macondoconfig.Config, gameEndedEvent *ipc.GameEndedEvent, gameId string) error {
 	// Josh, plz fix these asinine calls to incrementStatItem
 	events := history.GetEvents()
 
-	info := &IncrementInfo{Cfg: cfg,
+	info := &IncrementInfo{Ctx: ctx,
```

Comment:
In general you don't want to put context in structs. 

https://go.dev/blog/context-and-structs

However, this article highlights a case where you'd have to change a ton of functions and says it's probably ok to do that. I think as long as it's not abused it's probably ok.